asciidoc-common/install/snip-RHEL_repo_steps.adoc

```diff
@@ -0,0 +1,44 @@
+. Enable the required repositories:
+.. Register your system with the Content Delivery Network, entering your Customer Portal user name and password when prompted:
++
+[options="nowrap" subs="normal"]
+----
+# subscription-manager register
+----
++
+.. Find the `{enterprise-linux} Server` and `{virt-product-fullname}` subscription pools and record the pool IDs:
++
+[options="nowrap" subs="normal"]
+----
+# subscription-manager list --available
+----
++
+.. Use the pool IDs to attach the subscriptions to the system:
++
+[options="nowrap" subs="normal"]
+----
+# subscription-manager attach --pool=_poolid_
+----
++
+.. Configure the repositories:
++
+[options="nowrap" subs="normal"]
+----
+# subscription-manager repos \
+    --disable='*' \
+    --enable=rhel-7-server-rpms \
+    --enable=rhel-7-server-rhv-4-mgmt-agent-rpms \
+    --enable=rhel-7-server-ansible-2-rpms
```

Comment:
if this section is not already under conditional doc, rhel-7-server-rhv-4-mgmt-agent-rpms is not avilable with standard RHEL subscription so it's not available for ovirt users.

Reply:
This is actually conditionalized already, so it seems like that file is unneeded.